TrackDigi/TPCDigi/src/TPCDigiProcessor.cc

```diff
@@ -529,9 +529,9 @@ void TPCDigiProcessor::processEvent( LCEvent * evt )
         if( ptSqrdMC > (0.2*0.2) ) ++_NPhysicsAbove02GeVSimTPCHits ;
         if( ptSqrdMC > 1.0 )  ++_NPhysicsAbove1GeVSimTPCHits ;
         
-#ifdef DIGIPLOTS
-        if(_mcp) plotHelixHitResidual(_mcp, &thisPoint);
-#endif  
+      #ifdef DIGIPLOTS
+            if(_mcp) plotHelixHitResidual(_mcp, &thisPoint);
+      #endif  
```

Comment:
```suggestion
#ifdef DIGIPLOTS
        if(_mcp) plotHelixHitResidual(_mcp, &thisPoint);
#endif  
```
No unnecessary whitespace changes